src/components/dialogs/parameters/short-circuit-parameters.js

```diff
@@ -145,14 +145,6 @@ const BasicShortCircuitParameters = ({
             type: TYPES.bool,
             description: 'descWithFeederResult',
         },
-        studyType: {
-            type: TYPES.enum,
-            description: 'descStudyType',
-            values: {
-                TRANSIENT: 'descTransient',
-                SUB_TRANSIENT: 'descSubTransient',
```

Comment:
Suggestion: you can remove 'descTransient' and  'descSubTransient' from en.json and fr.json since we no longer use these variables in the project